FirebaseStorage/Sources/StorageComponent.swift

```diff
@@ -63,7 +63,7 @@ protocol StorageProvider {
 
   // MARK: - StorageProvider conformance
 
-  func storage(for bucket: String) -> Storage {
+  func storage(for bucket: String, app: FirebaseApp) -> Storage {
```

Comment:
There's a 1:1 relationship between a `StorageComponent` instance and `FirebaseApp` instance, so this shouldn't be necessary. as far as I can tell.

Reply:
This saves a force unwrap on the now optional `app`.

Reply:
Hmm, would `unowned` work in this case then?

Reply:
Good call! Looking promising in #11263